components/activities/EditActivityModal.tsx

```
@@ -0,0 +1,363 @@
+      ...prev,
+      details: {
+        ...prev.details,
+        [field]: value,
+      } as ActivityDetails,
+    }));
+  };
+
+  const renderDetailsFields = () => {
+    switch (formData.type) {
+      case 'feed':
+        const feedDetails = formData.details as FeedDetails;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)